src/index.html

```diff
@@ -58,80 +55,114 @@
     .col-xs-3 {
       border: none;
     }
-    .container h3 {
+
+    h3, h4 {
```

Comment:
This will make all h3 and h4 elements centered. For a larger application, I'd want something like that to be more context dependent (like "#content h3, #content h4" or something like that). For this application, for now, this should be fine.